### DISCUSSION TITLE
The Future of Mod

### DISCUSSION THREAD

**Discussion by LordMc47** [drgmes/doctor-who-mod#20] [Q&A] [answered]
Hi, I'm French so sorry in advance for my English. I really like your mod and honestly I have a lot of trouble projecting myself, even if there are plans for future updates. But I would like to know if you are still in development and if big updates are coming soon? Maybe I don't really know how to find out about this but I only found the GitHub... thank you in advance for your answer, I'm really looking forward to discovering the rest 🙏

**Comment by drgmes** [comment, marked as answer]
Hello, thank you for your message!

Yes, development continues, in the next week or two I plan to publish a new update, in which the system for rebuilding the main console room will be redesigned, added two new outer shells (and remade old Police Box) and the ability to change them through the TARDIS console unit

<details>
  <summary>New outer shells :)</summary>

  <img src="https://github.com/drgmes/doctor-who-mod/assets/33026426/7e844b5b-074e-4179-933c-a3f8f71cf944">
</details>

**Reply by LordMc47** [reply]
Glad to hear that I can't wait. Good luck with the development. I wouldn't hesitate to make suggestions if needed 🙏😁
